{"name": "Проверка неудачного обновления информации незалогиненного пользователя", "status": "passed", "description": "Проверка получения кода 401 Unauthorized и сообщения {\"success\": False, \"message\": \"You should be authorised\"} при отправке PATCH-запроса на обновление данных полей email, password, name незалогиненного пользователя", "parameters": [{"name": "update_field", "value": "'email'"}, {"name": "new_value", "value": "'natalya_novikov203@gmail.com'"}], "start": 1725094145788, "stop": 1725094146053, "uuid": "e0afa26a-6c2e-4b01-9ff7-1c07fde9be4e", "historyId": "771040b35142b38b11e32d9f53d3d1e7", "testCaseId": "6e21411b351c557c0b49423acf5a1aed", "fullName": "tests.test_change_user_data.TestsChangeUserData#test_update_data_without_auth_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_user_data"}, {"name": "subSuite", "value": "TestsChangeUserData"}, {"name": "host", "value": "MAIBENBEN-PC"}, {"name": "thread", "value": "3736-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_user_data"}]}